src/popmanager.cpp

```diff
@@ -118,4 +118,9 @@ bool PopManager::hasUncommittedChanges() const noexcept {
   return !uncommitted_.empty();
 }
 
+int PopManager::compareTwoBranches(const Chain<AltBlock>&,
+                                   const Chain<AltBlock>&) {
+  return 0;
+}
```

Comment:
Why this dummy function?

Reply:
I will implement fork resolution algorithm for the altcoin later, now I just add a signature of this function.